build-tools/config/webpack/webpack.partial.conf.devServer.js

```diff
@@ -0,0 +1,20 @@
+module.exports = (config, isDevelopment) => webpackConfig => ({
+  ...webpackConfig,
+  devServer: {
+    clientLogLevel: 'info',
+    historyApiFallback: true,
+    hot: true,
+    compress: true,
+    host: process.env.HOST || '0.0.0.0',
+    port: process.env.PORT || config.dev.port,
+    disableHostCheck: true,
+    open: false,
+    overlay: {
+      warnings: false,
+      errors: true,
+    },
+    proxy: config.dev.proxyTable,
+    // quiet: true,
```

Comment:
`quiet` can be removed, I can't see you want to have a quiet dev server running, or it should be configurable using environment variable. In that case a build server can enable this if needed.

Reply:
Then it should be un-commented right; as the default setup uses that plugin.

Reply:
Yes, but maybe it's off due [this bug](https://github.com/geowarin/friendly-errors-webpack-plugin/issues/71) I reported a while back

Reply:
Hehe makes sense then ;)

Reply:
Marking this as resolved, open a new ticket/pr when there are updates